tests/data/package_global_config_inputs_configuration_alerts/package/metadata/default.meta

```diff
@@ -0,0 +1,7 @@
+
+# Application-level permissions
+
+[]
+owner = admin
+access = read : [ * ], write : [ admin ]
```

Comment:
@arys-splunk should we allow write access to power users too?

Reply:
I think this should suffice as default permissions as I see a lot of TAs have their own customizations.